### PR TITLE
add style textcolor, size and weight

### PR DESCRIPTION
**Pull request by jameskarl133**
add style textcolor, size and weight

textColor: {
    color: 'white',
    fontSize: 24,
    fontWeight: 'bold'
  }

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -13,8 +13,13 @@ export default function App() {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
+    backgroundColor: 'green',
```
Comment (melbeno) [inline App.js]: change bg color to white

File: App.js [hunk 2]
```diff
@@ -13,8 +13,13 @@ export default function App() {
...
     alignItems: 'center',
     justifyContent: 'center',
   },
+  textColor: {
+    color: 'white',
```
Comment (melbeno) [inline App.js]: change textcolor to black

**Review by melbeno — CHANGES_REQUESTED** [review]
wapa toy

**Comment by jameskarl133** [comment]
unggoy!

**Review by melbeno — APPROVED** [review]
mana toy

**Pull request merged**